### PR TITLE
Dim disabled items in context menus

### LINKED ISSUE #1991: Search box weird font issue
**Describe the bug**
When you right click the search box in the search window, the font looks weird - this seems specific to the currently disabled options.

**To reproduce**
1. Ctrl + F in a channel
2. Right click in search box

**Screenshots**
image

**Chatterino version**
Chatterino Nightly 2.2.2 (commit 20e4d6b3d, 26.09.2020)

**Operating system**
Windows 10

### PR DESCRIPTION
**Pull request by Nerixyz**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description

The items in the context-menu, that shows up when right-clicking the input, all had the same text color. When right-clicking in any other input, the text had an "etched" text effect (#1991). This PR dims these disabled items and removes the text-effect. I have no clue why setting a `border` disables this text effect. There are [some references](https://www.qtcentre.org/threads/36417-disabled-QCheckBox-text-stylesheet-how-to-get-rid-of-change-color-of-etch-effect) to an [`etch-diabled-text`](https://bugreports.qt.io/browse/QTBUG-778), but that doesn't seem to (have) work(ed).

| Theme/State | White | Light | Dark | Black |
| --- | --- | --- | --- | --- |
| Before |  | chatterino_2023-03-01_19-33-31 | firefox_2023-03-01_19-30-50 | firefox_2023-03-01_19-33-14 |
| After | chatterino_2023-03-01_19-32-36 | chatterino_2023-03-01_19-32-23 | chatterino_2023-03-01_19-30-36 | firefox_2023-03-01_19-32-49 |

Fixes #1991.

### CODE REVIEW

**Review by Felanbird — APPROVED** [review]
Seems to work as I would expect it to 👍

**Comment by pajlada** [comment]
I imagine this should just be settable by the theme itself? now that themes are JSON-bound

remove draft & poke me when it's ready for review again and I can take a look

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +27,7 @@
 - Bugfix: Fixed Twitch channel-specific filters not being applied correctly. (#4529)
 - Bugfix: Fixed emote & badge tooltips not showing up when thumbnails were hidden. (#4509)
 - Bugfix: Fixed links with invalid IPv4 addresses being parsed. (#4576)
+- Bugfix: Fixed disabled items in context-menus having a weird or the default text color. (#4423)
```
Comment (Wissididom) [inline CHANGELOG.md:30]:
```suggestion
- Bugfix: Fixed disabled items in context-menus having a weird text-effect or the default text color. (#4423)
```
Not sure if that suggestion would be correct, but I wanted to point out that it seems to me that there is a word missing

**Review by Wissididom — COMMENTED** [review]
Just one thing I noticed

File: CHANGELOG.md (hunk 1 above)
Comment (Wissididom) [reply CHANGELOG.md:30]: When I think about it, it might even have been correct, if so, sorry

**Review by pajlada — CHANGES_REQUESTED** [review]
Regression: The "behind tab" background doesn't update automatically with this

**Review by pajlada — APPROVED** [review]
Works as expected on Linux

**Pull request merged**